packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/RepeatableComponent/index.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+import { Flex } from '@strapi/design-system/Flex';
+import { Box } from '@strapi/design-system/Box';
+import { Tooltip } from '@strapi/design-system/Tooltip';
+import { Typography } from '@strapi/design-system/Typography';
+import { Popover } from '@strapi/design-system/Popover';
+import { FocusTrap } from '@strapi/design-system/FocusTrap';
+import { SortIcon, stopPropagation } from '@strapi/helper-plugin';
+import { getTrad } from '../../../../utils';
+
+import CellValue from '../CellValue';
+
+const Button = styled.button`
+  svg {
+    > g,
+    path {
+      fill: ${({ theme }) => theme.colors.neutral500};
+    }
+  }
+  &:hover {
+    svg {
+      > g,
+      path {
+        fill: ${({ theme }) => theme.colors.neutral600};
+      }
+    }
+  }
+  &:active {
+    svg {
+      > g,
+      path {
+        fill: ${({ theme }) => theme.colors.neutral400};
+      }
+    }
+  }
+`;
+
+const ActionWrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: ${32 / 16}rem;
+  width: ${32 / 16}rem;
+  svg {
+    height: ${4 / 16}rem;
+  }
+`;
+
+const RepeatableComponentCell = ({ value, metadatas }) => {
+  const [visible, setVisible] = useState(false);
+  const { formatMessage } = useIntl();
+  const buttonRef = useRef();
+  const {
+    mainField: { type: mainFieldType, name: mainFieldName },
+  } = metadatas;
+  const subItems = [...value.slice(1)];
+
+  const handleTogglePopover = () => setVisible(prev => !prev);
+
+  return (
+    <Flex {...stopPropagation}>
+      <Tooltip
+        label={formatMessage({
+          id: getTrad('DynamicTable.component.repeatable.toggle'),
+          defaultMessage: 'Toggle repeatable values',
+        })}
+      >
+        <Button type="button" onClick={handleTogglePopover} ref={buttonRef}>
+          <Flex>
+            <Typography
+              style={{ maxWidth: '252px', cursor: 'pointer' }}
+              textColor="neutral800"
+              ellipsis
+            >
+              <CellValue type={mainFieldType} value={value[0][mainFieldName] || '[TBD]'} />
```

Comment:
Are we sure about the TBD? We are going to merge this PR on master soon.

WDYT about using the same placeholder we use in the LV temporarily? `-`

Reply:
Or we can always use the `id`

Reply:
+1 for the id

Reply:
I asked @maevalienard for an opinion and waiting for a response. I think `-` is too small to be clickable. I was thinking about "items" as fallback (similar to relations), but would like to discuss that with her first.

Reply:
Added in c98a4ef87679ed83615f03f8179400c9fef29e49. It seems like this is the default everywhere so far, so I guess it makes sense.

Reply:
Thanks @gu-stav, let's stick with that and see with the usage :D